### ISSUE TITLE
MonkeyNotify not working.

### ISSUE THREAD

**Issue by unamed000** [xamarin/ios-samples#470]

**Comment by rolfbjarne** [comment]
The notification showed up just fine for me after 20s. Can you try the latest versions and see if there's a difference? If it still fails, please paste your full version information (https://github.com/xamarin/xamarin-macios/wiki/Diagnosis#version-information)

**Comment by Pluxury** [comment]
@rolfbjarne  You're right the notification is displayed, but the `NotificationService.DidReceiveNotificationRequest` is not called. It is expected that the notification Titile will be changed and [modified] will be added to it

**Comment by rolfbjarne** [comment]
[Microsoft support for Xamarin](https://dotnet.microsoft.com/en-us/platform/support/policy/xamarin) will end on May 1, 2024 for all Xamarin SDKs. In preparation for this, all issues and PRs in this repository are being closed.
